vm/actor/src/lib.rs

```diff
@@ -30,16 +33,16 @@ lazy_static! {
 /// Number of token units in an abstract "FIL" token.
 /// The network works purely in the indivisible token amounts.
 /// This constant converts to a fixed decimal with more human-friendly scale.
-const TOKEN_PRECISION: u64 = 1_000_000_000_000_000_000;
+pub const TOKEN_PRECISION: u64 = 1_000_000_000_000_000_000;
```

Comment:
I know it isnt really part of this PR, but while youre at it, could you make this point to the one in types? (i think reward/types.rs also redefines this

Reply:
Yeah it was semi intentional since the go implementation redefines -- maybe they want the actors execution to not change even if the params outside changes? That was my thought, but it's not a good reason so I will share same